sql/catalyst/src/main/scala/org/apache/spark/sql/types/Metadata.scala

```diff
@@ -228,6 +232,8 @@ class MetadataBuilder {
     map ++= metadata.map
     this
   }
+  /** Puts a Long. */
```

Comment:
This comment is incorrect. Also, please leave an additional line of whitespace before this line.